R/AbstractGraphReporter.R

```diff
@@ -89,6 +89,7 @@ AbstractGraphReporter <- R6::R6Class(
                     searching = FALSE
                     , pageLength = 50
                     , lengthChange = FALSE
+                    , scrollX = TRUE
```

Comment:
@bburns632 @jameslamb 

Forgot I made this change, so I wanted to highlight to you. I _think_ the regular package reports somehow automatically had sideways scrollable wide tables, but the html_vignette ended up just having a table that overflowed out the side. This makes the vignette also behave that way, and I don't think it changes the behavior of the regular report.

Reply:
when would someone want this to be FALSE? Like can you explain why this needs to be in the interface as a lever you can pull instead of just one of those things we hard-code in as a sensible default behavior?

Reply:
I think you misunderstand the change @jameslamb. This is an option in the DT interface. The default by DT is no x-scrolling, which is what we have been using. I am proposing we change how our reports work by hard-coding in to _have_ x-scrolling. I am not exposing any new options to pkgnet interfaces.

Reply:
Ah! I looked at this too fast and from the way GitHub collapsed the code visually it looked like this was an arg being added to one of the public methods on this class. You're totally right, sorry!